src/core/qgsvectorlayer.cpp

```diff
@@ -3922,6 +3917,35 @@ QVariant QgsVectorLayer::aggregate( QgsAggregateCalculator::Aggregate aggregate,
   {
     return QVariant();
   }
+  bool hasFids = false;
+  QgsFeatureIds ids;
+  if ( fids )
+  {
+    qDebug() << 'fids';
+    ids = *fids;
+    hasFids = true;
+  }
+  else if ( context )
+  {
+    if ( context->indexOfScope( "Symbol scope" ) != -1 && mSymbolFeatureCounted )
+    {
+      qDebug() << "counted";
+      if ( mFeatureCounter )
+      {
+        qDebug() << "counter";
+        ids = mFeatureCounter->featureIds( context->variable( "symbol_id" ).toString() );
+        hasFids = true;
```

Comment:
@m-kuhn if you look at the debug log in travis or cdash, you will see that "counted" is printed, but the test on line 3933 never gets through and so the fids are never inputed. I can remove the check on line 3933 and you'll see that it segfaults.

Reply:
I will comment out this portion of the code, simply to have the test pass for the moment.